src/components/Button.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Proptypes from 'prop-types';
+import '../Button.css';
+
+const button = props => {
+  const { name, color } = props;
+  let { wide } = props;
+  wide = wide === true ? '50%' : '25%';
+  return (
+    <button type="button" className="Button" style={{ background: color, width: wide }}>{name}</button>
+  );
+};
```

Comment:
You can simplify this if you destructure the argument:

```js
const button = ({ name, color, wide }) => (
  <button
    type="button"
    className="Button"
    style={{ background: color, width: wide === true ? "50%" : "25%" }}
  >
    {name}
  </button>
);
```

Also, be aware that `defaultProps` will [eventually get deprecated](https://github.com/reactjs/rfcs/pull/107) for functional components, so a better approach in this particular case would be:

```js
const button = ({ name = "", color = "#FA9F42", wide = false }) => (
  <button
    type="button"
    className="Button"
    style={{ background: color, width: wide === true ? "50%" : "25%" }}
  >
    {name}
  </button>
);
```

Reply:
Thanks!